src/cli.rs

```diff
@@ -433,6 +447,9 @@ pub struct UpdateOpts {
     /// Print the diff, but don't write back the changes
     #[structopt(short = "n", long, global = true)]
     pub dry_run: bool,
+    /// Amount of concurrent updates that will be done at once
+    #[structopt(default_value = "5", long)]
+    pub conc_count: usize,
```

Comment:
```suggestion
    /// Maximum number of concurrent downloads
    #[structopt(default_value = "5", long)]
    pub max_concurrent_downloads: usize,
```

This is the name aria2 uses and I dislike it less than the current proposal, so let's take that instead. The flag may be a bit verbose, but given that most people will never need to change the default this should not be an issue.